setup/azurecmdfiles.wxi

```diff
@@ -3126,9 +3126,6 @@
                         <Component Id="cmp6DF3D322380ACA385B782FFE93501E46" Guid="*">
                             <File Id="filB160FE0B461B3EC308158442E13172D1" KeyPath="yes" Source="$(var.sourceDir)\ServiceManagement\Azure\HDInsight\Microsoft.WindowsAzure.Commands.ServiceManagement.Common.dll" />
                         </Component>
-                        <Component Id="cmp55C25F2FFCB6BB9E42E3D24DFF6408D1" Guid="*">
```

Comment:
It reports an error when have this file included and it does not exist in the build output directory. Error message is following:
  D:\p\azure-powershell-pr\setup\azurecmdfiles.wxi(3130): error LGHT0103: The system cannot find the file 'D:\p\azure-powershell-pr\setup\..\src\Package\Debug\ServiceManagement\Azure\HDInsight\Microsoft.WindowsAzure.Configuration.dll'. [D:\p\azure-powershell-pr\setup\az
urepowershell.wixproj]